### PR TITLE
Update Rules.txt

### PR DESCRIPTION
**Pull request by Colin-Tel**
## About the PR

Rules rule!!1

**Changelog**

:cl: Colin_Tel, DLondon
- tweak: Updated the server rules, check them out!

### CODE REVIEW

File: Resources/ServerInfo/Rules.txt [hunk 1]
```diff
@@ -81,7 +81,7 @@ The only exception to the above is theatrical roles such as the clown, boxer, an
     - Certain severe emergencies may make it reasonable for normal crew to defend themselves and their station, such as nuclear operatives, a zombie outbreak, space dragon attack, or other critical situation.
 
 [color=#a4885c]6.[/color] Follow powergaming guidelines:
-If you, in your given role, require an item that does not fall within your job’s usual parameters, have a valid reason to obtain and keep it. Finding something laying around without a clear owner is a valid reason.
+If you, in your given role, require an item that does not fall within your job’s usual parameters, you should have a valid reason to obtain and keep it. Finding something laying around without a clear owner is a valid reason.
```
Comment (deltanedas) [inline Resources/ServerInfo/Rules.txt:84]: doesnt this mean doctor finding insuls in maint is a valid reason, i.e. only stealing is covered which is basically selfantag

Comment (Colin-Tel) [reply Resources/ServerInfo/Rules.txt:84]: Yes, a doctor that finds insulated gloves in maints is not guilty of a rulebreak.

Comment (deltanedas) [reply Resources/ServerInfo/Rules.txt:84]: this rule was originally for people looting maints roundstart for gamer loot no, thats now explicitly allowed...

Comment (Colin-Tel) [reply Resources/ServerInfo/Rules.txt:84]: The way it was enforced in recent history has made rather innocuous activity subject to intrusive bwoinks from admins. Going forward, there isn't really anything wrong with going into maints and finding stuff.

Comment (IamVelcroboy) [reply Resources/ServerInfo/Rules.txt:84]: > The way it was enforced in recent history has made rather innocuous activity subject to intrusive bwoinks from admins. Going forward, there isn't really anything wrong with going into maints and finding stuff.

So, you're saying, "map more galoshes." Roger 👍😜

**Comment by ghost** [comment]
No shade Colin but I'm gonna have to close this because I copied this and put it in the PR that was just mentioned, sorgy. 😔

**Pull request closed without merging**